### DISCUSSION TITLE
Example for stopping Composition API watcher seems odd

### DISCUSSION THREAD

**Discussion by AloisSeckar** [vuejs/docs#2236] [Q&A] [answered]
I wanted to ask first, because maybe it is just me not understanding it, but I can't get into this sample code for [stopping a watcher](https://vuejs.org/guide/essentials/watchers.html#stopping-a-watcher) in Composition API:

![image](https://user-images.githubusercontent.com/7399922/219966975-1566e744-b834-45a7-8237-4e1c39389b71.png)

Either I don't need to instantiate `watchEffect` into variable `unwatch`, or I should later use it, shouldn't I? Something like `unwatch.unwatch()` or `unwatch(unwatch)`... Or is it really such a weird and unclear way of invoking it?

**Comment by skirtles-code** [comment, marked as answer]
I don't understand the question. `unwatch` is a function, you invoke it using `unwatch()`. The name `unwatch` is arbitrary, as it's just a local variable name being used to hold the function returned by `watchEffect`. I'm not sure why you think it's weird or unclear.

**Reply by AloisSeckar** [reply]
I guess it is because I am still not that proficient in JS world and having function as a return type, storing it into variable and then treat the variable as a function is sort of confusing for me.

Sorry for a "noob" question, would even delete it, but don't see such option here.
